doc/scapy/automotive_usage.rst

```diff
@@ -0,0 +1,387 @@
+****************
+Automotive usage
+****************
+
+.. note::
+    All automotive related features are only supported on linux systems, yet. CAN and ISOTP sockets in scapy3k are based on linux kernel modules.
```

Comment:
Hi !
When you copy/paste code, please check what you are doing :/
We're not in `scapy3k`